### ISSUE TITLE
Problems in nodule classification

### ISSUE THREAD

**Issue by NotMatter** [wentaozhu/DeepLung#52]
Hello, I used the ROI for classification for pure nodule classification task. The net used was “DPN92_3D”, the lr was 0.01(0-525 epochs), 0.001 (525-840), 0.0001(after 840) according your paper, respectively. After the running of “main_nodcls.py” (fold 5 was test fold), the loss (training and test) and accuracy (training and test) were shown in the follow. If overfitting occurred? Should I change the lr? Many thanks.
![image](https://user-images.githubusercontent.com/42600942/51009629-d2a24c80-158b-11e9-895a-62eb809f70c7.png)
![image](https://user-images.githubusercontent.com/42600942/51009639-df26a500-158b-11e9-962c-d2b57a380fea.png)

**Comment by NotMatter** [comment]
The accuracy(test and train) obtained was "acc = 100. * correct / total", not gbtteacc.

**Comment by wentaozhu** [comment]
It seems the results is fine because you can see the highest test accuracy is over 90%. You can check the results in Table 1 https://arxiv.org/pdf/1801.09555.pdf

**Comment by NotMatter** [comment]
> It seems the results is fine because you can see the highest test accuracy is over 90%. You can check the results in Table 1 https://arxiv.org/pdf/1801.09555.pdf

Thanks for your reply. The acc (90.44%) obtained in the paper was the average of 5 highest test acc?

**Comment by wentaozhu** [comment]
No. The reported accuracy is the average accuracy of the first 5 folds' results. For the result of each fold, we use the highest test accuracy which is the standard protocol of cross validation.
